include/aspect/introspection.h

```diff
@@ -465,6 +465,21 @@ namespace aspect
        * @}
        */
 
+      /**
+       * Return a vector with all used indices of base elements of the deal.II FiniteElement
+       * that are used for compositional fields. If several fields are the same type, they
+       * share base elements.
+      */
+      std::vector<unsigned int>
+      get_compositional_field_base_element_indices() const;
```

Comment:
Am I correct that the output array has length `# of compositional fields`? If so, would you mind adding that.

Reply:
No, currently it has length 1.